src/components/App.jsx

```diff
@@ -76,12 +76,9 @@ export class App extends Component {
               path="/permissions/data/:permission"
               element={<DataPermissions />}
             />
-            <Route
-              path="*"
-              element={
-                <Navigate to={isSmallView ? '/menu' : '/profile'} replace />
-              }
-            />
+            {isSmallView ? (
+              <Route path="*" element={<Navigate to={routes.menu} replace />} />
+            ) : null}
```

Comment:
nit: `{isSmallView && (<Route path="*" element={<Navigate to={routes.menu} replace />} />)}`

Reply:
This syntax creates bugs in react-native, that's why I took the habit of always returning null or string/element etc... but never undefined or boolean value. Do you think that's ok for us here to keep this syntax?

Reply:
si tu veux, c'est du nit, ça rajoute du bruit pour rien moi je trouve mais c'est du détail